core-bundle/contao/languages/de/default.xlf

```diff
@@ -2325,6 +2325,10 @@
         <source>Quick link</source>
         <target>Quicklink</target>
       </trans-unit>
+      <trans-unit id="MSC.lostPassword">
```

Comment:
German translations are managed on www.transifex.com, please don't add them to a pull request.